R/importWQP.R

```diff
@@ -145,6 +151,9 @@ importWQP <- function(obs_url, zip=TRUE, tz="UTC",
       attr(emptyReturn$ActivityStartDateTime, "tzone") <- tz
       attr(emptyReturn$ActivityEndDateTime, "tzone") <- tz
       
+      if(!convertType){
```

Comment:
I guess if convertType is true, we would want to make this empty data frame completely character

Reply:
Updated all columns to character data type within the  'emptyReturn' data frame.

Reply:
We don't want to make everything character unless the user choose convertType = FALSE. So we'll need to get the emptyReturn back to it's original types (numerics, dates, etc).

Then, you can add something like:
```r
if(!convertType){
  i <- sapply(emptyReturn, !is.character)
  emptyReturn[i] <- lapply(emptyReturn[i], as.character)
}
```
(you'll need to double-check that code....)